students/hyska_monika/lesson_06_dicts_tuples_sets_args_kwargs/Cubes.py

```diff
@@ -0,0 +1,52 @@
+"""
+Program use "set", print results for:
+intersection (A & B)
+relative complement of B in A (A - B)
+relative complement of A in B (B - A)
```

Comment:
The input is not conforming to the exercise, but I think you get the point of `set()` usage and if necessary you would implement it. So I just let it be.